### PR TITLE
Put back warning when rate limit has been reached on a specific updater but do not discard other updaters

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/repositories/updater.rs [hunk 1]
```diff
@@ -152,7 +152,19 @@ impl RepositoryStatsUpdater {
             // FIXME: The collect can be avoided if we use Itertools::chunks:
             // https://docs.rs/itertools/0.10.0/itertools/trait.Itertools.html#method.chunks.
             for chunk in needs_update.chunks(updater.chunk_size()) {
-                let res = updater.fetch_repositories(chunk)?;
+                let res = match updater.fetch_repositories(chunk) {
+                    Ok(r) => r,
+                    Err(err) => {
+                        if err.downcast_ref::<RateLimitReached>().is_some() {
+                            warn!(
+                                "rate limit reached, blocked the `{}` repository stats updater",
```
Comment (jyn514) [inline src/repositories/updater.rs:160]: "Blocked" doesn't seem correct, it will still collect stats the next time you run it. Maybe "ignored" is better?

Comment (GuillaumeGomez) [reply src/repositories/updater.rs:160]: I took the original message. Maybe "stopped"?

Comment (jyn514) [reply src/repositories/updater.rs:160]: "skipping" maybe - that seems to imply it's temporary.

Comment (GuillaumeGomez) [reply src/repositories/updater.rs:160]: Let's go for skipping then!

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by jyn514 — APPROVED** [review]

**Pull request merged**